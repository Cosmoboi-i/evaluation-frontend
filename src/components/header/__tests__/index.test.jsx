import React from "react";
import { render } from "@testing-library/react";
import Header from "..";

describe("Header", () => {
  it("should render properly", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
