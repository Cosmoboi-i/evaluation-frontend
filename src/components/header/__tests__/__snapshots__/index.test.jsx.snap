// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render properly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    hi
  </div>
</DocumentFragment>
`;
